src/st/no0/first_c_file.c

```diff
@@ -173,7 +173,37 @@ s16 func_us_801C1F98(Primitive* prim, s16 arg1) {
     return ret;
 }
 
-INCLUDE_ASM("st/no0/nonmatchings/first_c_file", func_us_801C2044);
+s16 func_us_801C2044(Primitive* prim, s16 offset) {
```

Comment:
Wasn't too sure what to call this, will try to decompile where it's called from and see if I can work out a good name